recipes/pyscrap/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: pyscrap
+  version: '0.0.9'
+
+source:
+  fn: pyscrap-0.0.9.tar.gz
+  url: https://pypi.python.org/packages/aa/cb/07ac450abb5c2eb7cdf52ceb394c8a35e4cca440e59e964a029d0696f181/pyscrap-0.0.9.tar.gz
```

Comment:
Please use jinja templating for the url. See the example.

Reply:
Thanks for the feedback @CJ-Wright , I appreciate it! I believe I fixed the things you suggested, but the build still fails. Can you help me with this when you have a chance please?